lib/lib-rds/README.md

```diff
@@ -0,0 +1,6 @@
+# @aws-sdk/lib-rds
+
+[![NPM version](https://img.shields.io/npm/v/@aws-sdk/lib-rds/latest.svg)](https://www.npmjs.com/package/@aws-sdk/lib-rds)
+[![NPM downloads](https://img.shields.io/npm/dm/@aws-sdk/lib-rds.svg)](https://www.npmjs.com/package/@aws-sdk/lib-rds)
+
+## Overview
```

Comment:
Since this package is expected to be consumed directly by the users, a README describing the interface is appreciated.